install/on_arch_linux.md

```diff
@@ -5,7 +5,7 @@ exclude: true
 permalink: /install/on_arch_linux/
 ---
 
-Arch Linux includes the Crystal compiler in the Community repository. You should also install `shards`, Crystal's dependency manager (see [The Shards command](../the_shards_command/README.md)).
+Arch Linux includes the Crystal compiler in the Community repository. You should also install `shards`, Crystal's dependency manager (see [The Shards command](/reference/the_shards_command/)).
```

Comment:
Links to reference and api should probably have the full path //crystal-lang.org/reference . In stg those links will break.